packages/camera/camera_web/CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
-## NEXT
+## 0.2.1+4
 
+* Internal migration from ui.hash* to Object.hash*
```

Comment:
`*` has special meaning in Markdown, so this isn't formatted correctly. You should use backticks on the method names anyway, which would avoid the issue.

Also, these need to follow the style guide linked from the checklist. It's missing the period at the end, and it should ideally be "Migrates from" rather than "Internal migration from".

So:
```
* Migrates from `ui.hash*` to `Object.hash*`.
```

Reply:
done! thanks :)